packages/shared/src/components/layout/common.tsx

```diff
@@ -94,8 +99,13 @@ export const SearchControlHeader = ({
     SharedFeedPage.CustomForm,
   ];
 
+  const shieldBtnProps: ButtonProps<'button'> = customFeedPlacement && {
+    size: ButtonSize.Small,
+    variant: ButtonVariant.Tertiary,
+  };
```

Comment:
This will return `false` if not valid. We'll have to use ternary to specify null or undefined. Otherwise, the destructure from the component will break.
```suggestion
  const shieldBtnProps: ButtonProps<'button'> = customFeedPlacement ? {
    size: ButtonSize.Small,
    variant: ButtonVariant.Tertiary,
  } : undefined;
```